Gemfile.lock

```diff
@@ -49,16 +32,16 @@ GEM
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.12.0)
     rspec-support (3.12.0)
-    ruby2_keywords (0.0.5)
+    uri (0.13.1)
 
 PLATFORMS
   ruby
-  x86_64-linux
+  x86_64-darwin-22
```

Comment:
I have no idea if this matters, I haven't used Ruby in so long, just wanted to flag that this has changed from Linux to Mac.

Reply:
good catch! I'm reverting this just to be safe.